specs/_features/maxeb_increase/capella.py

```diff
@@ -656,6 +658,7 @@ class BeaconState(Container):
     # Registry
     validators: List[Validator, VALIDATOR_REGISTRY_LIMIT]
     balances: List[Gwei, VALIDATOR_REGISTRY_LIMIT]
+    exit_queue_churn: Gwei
```

Comment:
I'm expecting exits and activations to be symmetric. If we need `exit_queue_churn` don't we also need `activation_queue_churn`? :)

Reply:
@fradamt wrote this part, but IIUC the reason we need this is because we need to preserve the state of the amount of Gwei in the exit_queue epoch has been consumed. for the activation queue, we can just use a constant and fill up as many validators as fit, without worrying about a validator being split over multiple epochs.

Reply:
This variable represent the excess churn of the current highest exit_epoch in the validators records. I think the current name is misleading since exit_queue_churn exist semantically in the codebase and refers to a different thing. I would add `excess` in the name to clarify that difference
```suggestion
    exit_queue_excess_churn: Gwei
```